### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by IgorOlifirenko**

### CODE REVIEW

**Review by Oleksandr563487953 — APPROVED** [review]

File: tests/test_restore_names.py [hunk 1]
```diff
@@ -1,3 +1,36 @@
 from app.restore_names import restore_names
 
-# write your tests here
+
+def test_should_return_none():
+    assert restore_names([]) is None
+
+
+def test_first_name_is_none_or_first_name_is_lost():
```
Comment (IvanRamyk) [inline tests/test_restore_names.py:8]: Please, add separate tests for both cases (None and missing key).
In that case, it'd be easier to find where a bug has occurred if the function does something wrong

**Review by IvanRamyk — CHANGES_REQUESTED** [review]
